Api/OneDrive/OneDriveProvider.cs

```diff
@@ -240,12 +242,5 @@ public override string Email
 				return email;
 			}
 		}
-
-		public override async Task Logout()
-		{
-			Api.ResetData();
-			await RemoveApi(Id);
-			ApiManager.Shared.SaveApi(Api);
```

Comment:
As long as the logout, is called from https://github.com/Clancey/gMusic/blob/cbb60e37b20e2fd0a321dd42ba7b42b046a8147f/MusicPlayer.Shared/Managers/ApiManager.cs#L288 it should be fine. The reason is to make sure it isnt showing up in the API manager anymore for logging in purposes.